packages/gradient_boosting_model/setup.py

```diff
@@ -0,0 +1,78 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+import io
+import os
+
+from setuptools import find_packages, setup
+from gradient_boosting_model import config, PACKAGE_ROOT
+
+
+# Package meta-data.
+NAME = config['package_name'].data
+DESCRIPTION = "Train and deploy LARS Lasso regression model."
```

Comment:
what is LARS? should be removed I think

Reply:
Also, is this for the logit, or the gbm? I am confused now, are we building both models? one?

Reply:
gbm. Fixed